src/control/cmd/dmg/acl.go

```diff
@@ -44,6 +44,12 @@ func readACLFile(aclFile string) ([]string, error) {
 	return parseACL(file)
 }
 
+// isACLFileComment checks whether the line is formatted as a comment for an
+// ACL file.
+func isACLFileComment(line string) bool {
+	return strings.HasPrefix(line, "#")
```

Comment:
Do you need to trim the whitespace at the front of the line? Would this handle " #"?